tox.ini

```diff
@@ -4,17 +4,20 @@ skipsdist = true
 
 [testenv]
 deps=
+    aniso8601
     pytest>=2.7.2
     graphql-core>=1.1
     promise>=2.0
     graphql-relay>=0.4.5
     six
     blinker
     singledispatch
+    snapshottest
     mock
     pytz
     iso8601
     pytest-benchmark
+    pytest-mock
```

Comment:
Why do you have to define the dependencies here when they are already defined in the `setup.py`?

Reply:
This was the existing pattern in this repo; tox is pulling deps only from that list for running tests and not reading the list of test reqs from setup.py. 

I suggest accepting this PR so tox stops failing for now. Then I'd be happy to put out a followup PR to create a requirements-dev.txt file which can be the single source of truth for test requirements.

Reply:
But why doesn't the tox file just have the `pip install -e ".[test]"` command that is outlined in the README? Then the whole `[testenv]` section just becomes:

```
[testenv]
commands = 
	pip install -e ".[test]"
	py.test --cov=graphene graphene examples
```

and there is no need to define requirements in multiple places.

Reply:
Sounds good; updated!